app/controllers/AdminController.scala

```diff
@@ -678,52 +678,24 @@ class AdminController @Inject() (implicit val env: Environment[User, SessionAuth
       "total" -> StreetEdgeTable.countTotalStreets(),
       "audited" -> Json.obj(
         "all_users" -> Json.obj(
```

Comment:
How do we feel about this JSON structure? I think previously we had it so that the all_users total and high quality total were on the same level as the registered, anon, etc. JSON sub-objects. I feel like this makes a little more sense since it's pretty much row by row as it is in the table in the admin page, but let me know if you want me to structure it like that instead!

Reply:
Yeah I think that's fine! I was initially thinking that it doesn't make much sense to have an "all users" object, and then within that object we have one property for all users and another for a subset of users :joy: But now that the whole JSON object is smaller and less overwhelming, it feels less confusing